### ISSUE TITLE
Add instructions on how to completely remove Leapp

### ISSUE THREAD

**Issue by zackriso** [Noovolari/leapp#329]
In some scenarios, it might be very useful to downgrade or try a previous version of Leapp, until a bug is fixed. 

In that case, instructions on how to properly uninstall Leapp is extremely important and useful. 

Can we please add removal instructions in the documentations?

**Comment by c-sami** [comment]
That's very true. 
Information about where Leapp will install application, configuration and log files can be found in the documentation under "Troubleshooting - Application data" (https://docs.leapp.cloud/0.14.2/troubleshooting/app-data/), but there could be also sub-page how to uninstall all traces of the application manually.

**Comment by andreacavagna01** [comment]
I agree @zackriso, maybe @ericvilla can help us in describing how to uninstall the App
